spatialdata/_core/elements.py

```diff
@@ -48,26 +51,46 @@ def shape(self) -> Tuple[int, ...]:
 
 
 class Image(BaseElement):
-    def __init__(self, image: DataArray, transform: Transform) -> None:
-        self.data: DataArray = image
+    def __init__(self, image: DataArray, transform: BaseTransformation) -> None:
+        if isinstance(image, DataArray):
+            self.data = image
+        elif isinstance(image, np.ndarray):
+            self.data = DataArray(image)
+        elif isinstance(image, dask.array.core.Array):
+            self.data = DataArray(image)
+        else:
+            raise TypeError("Image must be a DataArray, numpy array or dask array")
         self.transforms = transform
+        self.axes = self._infer_axes(image.shape)
         super().__init__()
 
     @staticmethod
     def parse_image(data: Any, transform: Optional[Any] = None) -> "Image":
-        data, transform = parse_dataset(data, transform)
+        # data, transform = parse_dataset(data, transform)
+        if transform is None:
+            transform = Identity()
         return Image(data, transform)
 
     def to_zarr(self, group: zarr.Group, name: str, scaler: Optional[Scaler] = None) -> None:
-        # TODO: write coordinate transforms
         # TODO: allow to write from path
+        assert isinstance(self.transforms, BaseTransformation)
+        coordinate_transformations = self.transforms.to_dict()
+        # at the moment we don't use the compressor because of the bug described here (it makes some tests the
+        # test_readwrite_roundtrip fail) https://github.com/ome/ome-zarr-py/issues/219
         write_image(
             image=self.data.data,
             group=group,
-            axes=["c", "y", "x"],  # TODO: infer before.
+            axes=self.axes,
             scaler=scaler,
+            coordinate_transformations=[[coordinate_transformations]],
+            storage_options={"compressor": None},
```

Comment:
shouldn't this be exposed, even if just by kwargs?

Reply:
Yes, and also `chunks` needs to be exposed. It's crucial to let the user choose a good chunk size (we can also make a good guess) to have good performance also for massive images.

Reply:
✔️ exposed `compressor` and `chunks`